client/state/privacy-policy/schema.js

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+export const privacyPolicySchema = {
+	type: 'object',
+	additionalProperties: false,
+	patternProperties: {
+		'^[a-z/_-]+$': {
```

Comment:
Why such a generic schema? It looks like there are the following keys. Of those we can probably specify whatever ones the client actually uses.

```
created
content
id
modified
notification_period
raw_content
title
```

Reply:
👍